docs/_components/badges.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Badges
+lead: >
+  Badges are used to encourage user behavior throughout the site.
+---
+
+Badges can help signal to the user that they had completed a workflow desired by Login.gov. Primarily, these are used on our user account page to encourage more secure practices.
+
+Badges are comprised with a div using the `lg-verification-badge` class and a small SVG image alongside a `span` of text.
+
+## An unphishable badge example
+
+The unphishable badge is used to indicate that an account is only using security keys as its MFA method.
+
+{% capture example %}
+<div class="lg-verification-badge">
```

Comment:
I restructured this to be an img and span inside a div to make it easier to use the `vertical-alignment` controls.

Reply:
ah that makes sense